### ISSUE TITLE
Initiative: homepage launch

### ISSUE THREAD

**Issue by ExperimentsInHonesty** [hackforla/website#2481]
### Overview
We need to track and support the final wrap up items for the homepage launch, so that it can be completed.

#### Details
The homepage launch is overdue by at least a year, because we have not been actively monitoring its final issues.

### Action Items
- [x] List all the areas that will need to be in good shape before the homepage launch
- [x] Identify what needs to be done to fix the area
- [x] Link any issues reference in the this epic
- [x] Make new Epic issues for each section and link them in the epics and below 
   - [x] Add a link to each epic below
- [ ] Review each epic and add issues with label
- [ ] Classify into MoScoW
- [ ] Updating all epic issues with template changes
- [ ] Review all epic, and determine if all the issues needed for the epic have been created
   - [ ] if it needs issues 
      - define new issues to create
      - create issues
   - [ ] When all the epics have all issues
      - review the status of the issues
         - resolve blockers
         - refine monitoring cadence
      - [ ] estimate time to completion across all

#### Final tasks once issue is completed
- After the website homepage launch, research will look into a global change for all the menu items and their corresponding page/section title.  See [DR: Capitalization of menu items in nav bar and header](https://github.com/hackforla/website/wiki/DR:-Capitalization-of-menu-items-in-nav-bar-and-header)

### Resources/Instructions
<details><summary>Desktop</summary>

<img src="https://user-images.githubusercontent.com/40401149/144154190-da9bdca7-11bb-4960-9da3-7b132678ea08.png">
</details> 

<details><summary>Mobile</summary>

<img src="https://user-images.githubusercontent.com/40401149/144154195-867292e4-e034-4631-8d3f-f1bfb05dae14.png">
</details> 

- Spreadsheet [Hompage epic priority](https://docs.google.com/spreadsheets/d/1Ima7-nCQ3qodsuB3--efK8bru5UBzs2LkOwH6NxPBSg/edit?gid=0#gid=0)
#### Issues created 
- [ ] #7664 
- [ ] #7666
- [ ] #7667
- [ ] #7671
- [ ] #7672
- [ ] #7674
- [ ] #7676
- [ ] #7679
- [ ] #7904
- [ ] #8417
- [ ] #8472

#### Epics that need issues to be made
- [ ] #7662
   - [x] #7663
- [ ] #7670
- [ ] #7673
- [ ] #7675
- [x] #7678
- [x] #7680
   - [ ] #7677
#### Epics that have all issues made

---
### Additional links
<details><summary>Homepage image</summary>
<p>

![Group 737](https://github.com/hackforla/website/assets/37763229/7c72b1c2-e7f6-4494-9834-3e7b9951ffac)

**Comment by ExperimentsInHonesty** [comment]
- [ ] About page - Bonnie to define updates to about page
  - [ ] Letter from ED #860
  - [x] Accomplishments update #2585 
    - [ ] Figure out how we are going to display 2020 and 2021 accomplishments on the page
      - https://github.com/hackforla/website/blob/gh-pages/_data/internal/accomplishments.yml  (has the 2020 content in it)
      - https://github.com/hackforla/website/blob/gh-pages/_includes/about-page/about-card-accomplishments.html controls how it appears in the accomplishments section
        - [x] Have asked Phu to make an issue for design to reimagine this section.
  - [ ] SDGs list from projects #2037
    - [ ] Saumil will make an issue for adding the sdg info to the site (small issues)
  - [x] Formatting of press section is not responsive #1498 
- [ ] Dashboard 
  - [ ] #2776
  - [ ] add Google Analytics to the dashboard to show how many people we reach (Could Have)
- [ ] Program Areas Pages
  - [ ] - Program areas page mini cards need to be delivered dynamically so that they match the results from the project filters. #2485 
- [ ] Projects Technologies being up to date
  - [x] - We need the technologies field to be free of languages https://www.hackforla.org/projects/?technologies=Python,SASS,SCSS,TypeScript
  - [x] D on above - add languages to the technologies filter so that they can appear together change title of filter to Technologies / Languages 
  - [ ] Create a second project page for checking quality of lang vs tech, and it just lists all the projects and has those two filters separated. #2502 
- [ ] Communities of practice page up to date 
  - [x] Needs to have github for all CoPs https://github.com/hackforla/website/issues/2357#issuecomment-968349565
  - [ ] Needs to be reviewed to see if updates are required
- [ ] [Join us page issues](https://github.com/hackforla/website/issues?q=is%3Aopen+is%3Aissue+label%3A%22P-Feature%3A+Join+Page%22)
- [ ] wins - see more issue not resolved
  - [ ] https://github.com/hackforla/website/issues/2339
- [ ] https://www.hackforla.org/about/#news
- [ ] in order for the footer to work, there needs to be anchor tags on the join us page https://www.hackforla.org/join#volunteer
  - [ ] https://github.com/hackforla/website/issues/3026
- [ ] revise order on footer to match the join page #2958 
- [ ] ~ask Sarah to~ make an anchor to the contact us section of the homepage: https://github.com/hackforla/website/issues/3023
- [ ] make a new issue for revising the events page --> The issue is written and this is the issue: https://github.com/hackforla/website/issues/3027
- [ ] Toolkit gets removed from site for launch if not ready
  - [ ] https://github.com/hackforla/website/issues/2802
- [ ] update site map
- [ ] https://github.com/hackforla/website/issues/3377
- [x] https://github.com/hackforla/website/issues/2211


### Post launch
- [ ] Wins randomizer (has not yet been added to about)
  - [x] https://github.com/hackforla/website/issues/2304
  - [ ] will need to add it to the page after homepage launch (its already coded for the homepage but stuck on the branch).
  - [ ] will need to add code to about page

**Comment by JessicaLucindaCheng** [comment]
- [x] Is Contact Us is supposed to go to the Get In Touch section of redesigned homepage? Yes
<details>
  <summary>Get In Touch</summary>
  <img width="949" alt="get-in-touch" src="https://user-images.githubusercontent.com/31293603/161420525-bb555ede-ec94-479b-83a2-95b9c6b60c55.png">
</details>

- [ ] Differences between #2455 and #2393. We should be consistent. Which do you prefer? Probably title capitialization but reach out to zzhoje about the discrepancy to confirm.

  - [ ] `Submit your pitch` vs `Submit your Pitch` (nav bar vs footer)
  - [ ] `Get in touch` vs `Get in Touch` (nav bar vs footer)

<details>
  <summary>Header Nav Bar (partial)</summary>
  <img width="272" alt="submit-your-pitch-and-get-in-touch-in-nav-header" src="https://user-images.githubusercontent.com/31293603/161420415-2c08c720-7ebc-40b6-8c0e-41701f0d4a00.png">
</details>

<details>
  <summary>Footer</summary>
  <img width="843" alt="submit-your-pitch-and-get-in-touch-in-footer" src="https://user-images.githubusercontent.com/31293603/161420420-806f16f4-0caa-48e9-86cd-3156d2747dcf.png">
</details>

**Comment by JessicaLucindaCheng** [comment]
@zzhoje I noticed discrepancies in the capitalization of certain words in the menu design for our navigation bar header compared to the menu design for the footer. 
<details>
  <summary>Partial Screenshot of Header Nav Bar Design</summary>
  <img width="272" alt="submit-your-pitch-and-get-in-touch-in-nav-header" src="https://user-images.githubusercontent.com/31293603/161420415-2c08c720-7ebc-40b6-8c0e-41701f0d4a00.png">
</details>

<details>
  <summary>Screenshot of Footer Design</summary>
  <img width="843" alt="submit-your-pitch-and-get-in-touch-in-footer" src="https://user-images.githubusercontent.com/31293603/161420420-806f16f4-0caa-48e9-86cd-3156d2747dcf.png">
</details>

However, I think the capitalization should be the same for the header and footer menus. Thus, I just want to confirm with you the following:
  - There is a difference in capitalization for the word pitch. Should it be `Submit your pitch` or `Submit your Pitch`?
  - There is a difference in capitalization for the word touch. Should it be `Get in touch` or `Get in Touch`?

Thank you.

**Comment by zzhoje** [comment]
@JessicaLucindaCheng I think it would make more sense for these navigation menu items to be in title case, which it seems like the footer is already going for. In title case, the two examples you pulled would be “Submit Your Pitch” and “Get in Touch”. I just think it will be easier for us to keep all of the titles/headers/footers/buttons consistent this way, and title case tends to give more symmetry and visual prominence. One way to check if a phrase is in title case is by using https://capitalizemytitle.com/ . Please let me know what you think! Thanks!

**Comment by SAUMILDHANKAR** [comment]
> I noticed discrepancies in the capitalization of certain words in the menu design for our navigation bar header compared to the menu design for the footer. 
> <details>
>   <summary>Partial Screenshot of Header Nav Bar Design</summary>
>   <img width="272" alt="submit-your-pitch-and-get-in-touch-in-nav-header" src="https://user-images.githubusercontent.com/31293603/161420415-2c08c720-7ebc-40b6-8c0e-41701f0d4a00.png">
> </details>
> 
> <details>
>   <summary>Screenshot of Footer Design</summary>
>   <img width="843" alt="submit-your-pitch-and-get-in-touch-in-footer" src="https://user-images.githubusercontent.com/31293603/161420420-806f16f4-0caa-48e9-86cd-3156d2747dcf.png">
> </details>
> 
> However, I think the capitalization should be the same for the header and footer menus. Thus, I just want to confirm with you the following:
>   - There is a difference in capitalization for the word pitch. Should it be `Submit your pitch` or `Submit your Pitch`?
>   - There is a difference in capitalization for the word touch. Should it be `Get in touch` or `Get in Touch`?
> 
> Thank you.

Decision made by Bonnie: It is going to be "Submit your Pitch" and "Get in Touch"

After the website homepage launch, research will look into a global change for all the menu items and their corresponding page/section titles.

**Comment by JessicaLucindaCheng** [comment]
> Decision made by Bonnie: It is going to be "Submit your Pitch" and "Get in Touch"
> 
> After the website homepage launch, research will look into a global change for all the menu items and their corresponding page/section titles.

DR made for this decision: [DR: Capitalization of menu items in nav bar and header](https://github.com/hackforla/website/wiki/DR:-Capitalization-of-menu-items-in-nav-bar-and-header)

Not sure if this was passed on to research: "After the website homepage launch, research will look into a global change for all the menu items and their corresponding page/section titles."

**Comment by abdiaz2018** [comment]
- All overview links work correctly. #2211

**Comment by ExperimentsInHonesty** [comment]
- [ ] About page - Bonnie to define updates to about page
  - [ ] Letter from ED #860
  - [x] Accomplishments update #2585 
  - [ ] Figure out how we are going to display 2020 and 2021 accomplishments on the page #3035
      - https://github.com/hackforla/website/blob/gh-pages/_data/internal/accomplishments.yml  (has the 2020 content in it)
      - https://github.com/hackforla/website/blob/gh-pages/_includes/about-page/about-card-accomplishments.html controls how it appears in the accomplishments section
        - Waiting on content for acomplishments
  - [ ] SDGs list from projects #2037
    - [ ] Saumil will make an issue for adding the sdg info to the site (small issues)
  - [x] Formatting of press section is not responsive #1498 
- [ ] Dashboard 
  - [ ] #2776
  - [ ] add Google Analytics to the dashboard to show how many people we reach (Could Have)
- [x] Program Areas Pages
  - [x] - Program areas page mini cards need to be delivered dynamically so that they match the results from the project filters. #2485 
  - [ ] Update all the pages that list Diversity, Equity & Inclusion incorrectly
    - [ ] https://github.com/hackforla/website/issues/4284
    - [ ] https://github.com/hackforla/website/issues/4286
    - [ ] https://github.com/hackforla/website/issues/4287
- [ ] Projects Technologies being up to date
  - [x] - We need the technologies field to be free of languages https://www.hackforla.org/projects/?technologies=Python,SASS,SCSS,TypeScript
  - [x] D on above - add languages to the technologies filter so that they can appear together change title of filter to Technologies / Languages 
  - [x] Create a second project page for checking quality of lang vs tech, and it just lists all the projects and has those two filters separated. #2502 
- [x] Communities of practice page up to date 
  - [x] Needs to have github for all CoPs https://github.com/hackforla/website/issues/2357#issuecomment-968349565
  - [x] https://github.com/hackforla/website/issues/4239
  - [x] Needs to be reviewed to see if updates are required
- [x] Wins randomizer
  - [x] https://github.com/hackforla/website/issues/2304
- [x] [Join us page issues](https://github.com/hackforla/website/issues?q=is%3Aopen+is%3Aissue+label%3A%22P-Feature%3A+Join+Page%22)
- [x] wins - see more issue not resolved
  - [x] https://github.com/hackforla/website/issues/2339
- [x] https://www.hackforla.org/about/#news
- [x] in order for the footer to work, there needs to be anchor tags on the join us page https://www.hackforla.org/join#volunteer
  - [x] https://github.com/hackforla/website/issues/3026
- [x] revise order on footer to match the join page #2958 
- [ ] ~ask Sarah to~ make an anchor to the contact us section of the homepage: https://github.com/hackforla/website/issues/3023
- [x] make a new issue for revising the events page --> The issue is written and this is the issue: https://github.com/hackforla/website/issues/3027
- [x] Toolkit gets removed from site for launch if not ready
  - [x] https://github.com/hackforla/website/issues/2802
- [x] update site map
- [ ] https://github.com/hackforla/website/issues/3377
- [x] https://github.com/hackforla/website/issues/2211

**Comment by ExperimentsInHonesty** [comment]
Features that need to be finished or parts finished

### About Us page
- URL: https://hackforla.org/about
- [About Us issues on project board](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22p-feature%3A+about+us%22)
- Sections
   - ED Letter 
     - [ ] #860 
   - Sustainability
     - See SDG section below, the content for this, should be dynamic and be pulled from each project file.
   - Accomplishments 
      - [ ] Find all the other accomplishment content that has been written and identify what is missing
        <details><summary>URLs for technical issues</summary>
        <p>
           - https://github.com/hackforla/website/blob/gh-pages/_data/internal/accomplishments.yml (has the 2020 content in it)
           - https://github.com/hackforla/website/blob/gh-pages/_includes/about-page/about-card-accomplishments.html controls how it appears in the accomplishments section     
        </p>
        </details> 
   - Metrics (dashboard)
      - URL: https://hackforla.org/dashboard
      - [Org Dashboard Issues on the project board](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22p-feature%3A+dashboard%22)
      - [ ] Review survey results and identify what might need to be changed
      - [ ] Add Google Analytics to the Dashboard https://github.com/hackforla/website/issues/4537
   - Partners
      - [ ] Change title to be Current and Past Partners
   - Sponsors
      - [ ] Change title to be Current and Past Sponsors

### Wins
- URL: https://hackforla.org/wins
- [ ] [Wins issues on the project board](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22p-feature%3A+wins+page%22)
   - [ ] Wins Randomizer
- [ ] [Dashboard](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22p-feature%3A+dashboard%22)
   - [ ] add Google Analytics to the dashboard to show how many people we reach (Could Have)

### Join Us
- [ ] [Join us page Issues on the project board](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22p-feature%3A+Join+Page%22)

### Program Areas
- [ ] URL: https://www.hackforla.org/program-areas
- [ ] [Program Areas Issues on the project board](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22p-feature%3A+program+area%22)


### Projects Page
- [ ] URL: https://www.hackforla.org/projects
- [ ] [Projects page Issues on the project board](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22p-feature%3A+projects+page%22)
  - The drop downs have to have consistent content in them (problems with languages, technologies and tools).  
     - https://github.com/hackforla/website/issues/6709
     - https://github.com/hackforla/website/issues/6327
     - https://github.com/hackforla/website/issues/6138
     - https://github.com/hackforla/website/issues/4593
     - https://github.com/hackforla/website/issues/4434
     - https://github.com/hackforla/website/issues/4262
  - They need to look better.  The chevron we currently have is unattractive.  See http://civictechindex.org/about/faq and make an ER to improve ours
  - Make an ER to address the difference between
     - https://www.hackforla.org/projects-check/?technologies=Docker
     - https://www.hackforla.org/projects-check/?languages=Dockerfile

### CoPs
- [ ] URL: https://www.hackforla.org/communities-of-practice
- [ ] [Communities of Practice Issues on the project board](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22p-feature%3A+communities+of+practice%22)
  - [ ] Needs to be reviewed to see if updates are required

### Events
- URL: https://www.hackforla.org/events
- [ ] [Events Issues on the project board](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22p-feature%3A+events%22)

### Toolkit
- URL: https://www.hackforla.org/toolkit
- [ ] Remove the toolkit from the menu.  Bring it back when its ready

### Sitemap
- URL: https://www.hackforla.org/sitemap
- [ ] find or make new issue to update sitemap

### On many pages
#### SDGs
- [SDG issues on project Board](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22p-feature%3A+sdgs%22)
- Issues
   - https://github.com/hackforla/website/issues/4229
   - https://github.com/hackforla/website/issues/2037
- Resources
   - [Program Area Audit Spreadsheet](https://docs.google.com/spreadsheets/d/1wkzmWck-aGTzGxiryYEUhGbKRJX8rDDT3RLsAn-E-9o/edit#gid=0) - has the SDG info for all projects and issue tracking

### Sitewide
- [ ] https://github.com/hackforla/website/issues/6140
- all spelling issues
   - https://github.com/hackforla/website/issues/5450
- [ ] confirm that all links on the header and footer work

### Additional links
<details><summary>Homepage image</summary>
<p>

![Group 737](https://github.com/hackforla/website/assets/37763229/7c72b1c2-e7f6-4494-9834-3e7b9951ffac)


</p>
</details>

**Comment by ExperimentsInHonesty** [comment]
- Finish going over the Jan 11th post (items below) and make sure there is an issue or ER for each item mentioned
- Add the rest of the features that are important to the launch to the 2023-03-27 post
- Review the board for each feature area and identify any items still missing
  - make ERs for missing items
- Weekly check the pages to see and report progress (i.e., none of the outstanding issues or ERs are blockers to launching, or X number of points for this to be ready)

---

- [ ] Projects Technologies being up to date
  - [x] - We need the technologies field to be free of languages https://www.hackforla.org/projects/?technologies=Python,SASS,SCSS,TypeScript
  - [x] D on above - add languages to the technologies filter so that they can appear together change title of filter to Technologies / Languages 
  - [x] Create a second project page for checking quality of lang vs tech, and it just lists all the projects and has those two filters separated. #2502 
- [ ] https://www.hackforla.org/about/#news
- [ ] in order for the footer to work, there needs to be anchor tags on the join us page https://www.hackforla.org/join#volunteer
  - [ ] https://github.com/hackforla/website/issues/3026
- [ ] revise order on footer to match the join page #2958 
- [ ] ~ask Sarah to~ make an anchor to the contact us section of the homepage: https://github.com/hackforla/website/issues/3023
- [ ] Toolkit gets removed from site for launch if not ready
  - [ ] https://github.com/hackforla/website/issues/2802
- [ ] update site map
- [ ] https://github.com/hackforla/website/issues/3377
- [x] https://github.com/hackforla/website/issues/2211

**Comment by ExperimentsInHonesty** [comment]
This is the latest list audited and expanded from these prior comments:
- https://github.com/hackforla/website/issues/2481#issuecomment-1484644038
- https://github.com/hackforla/website/issues/2481#issuecomment-1484680290

This is a list of all the issues related to homepage launch. There may need to be more issues made (see each section)
---

### About Us page
- URL: https://hackforla.org/about
- [About Us issues on project board](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22p-feature%3A+about+us%22)
- Sections
   - ED Letter 
     - [x] Revise the letter in this closed issue #860
     - [x] make a new issue to add the letter to the site.
   - Sustainability
     - See SDG section below, the content for this, should be dynamic and be pulled from each project file.
   - Accomplishments 
      - [x] Find all the other accomplishment content that has been written and identify what is missing
        <details><summary>URLs for technical issues</summary>
        <p>
           - https://github.com/hackforla/website/blob/gh-pages/_data/internal/accomplishments.yml (has the 2020 content in it)
           - https://github.com/hackforla/website/blob/gh-pages/_includes/about-page/about-card-accomplishments.html controls how it appears in the accomplishments section     
        </p>
        </details> 
      - [x] See Wins section below for the wins randomizer issue
   - Metrics (dashboard)
      - URL: https://hackforla.org/dashboard
      - [Org Dashboard Issues on the project board](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22p-feature%3A+dashboard%22)
      - [x] Review survey results and identify what might need to be changed
      - [x] Add Google Analytics to the Dashboard https://github.com/hackforla/website/issues/4537
      - [x] See this issue to evaluate if its related https://github.com/hackforla/website/issues/1730
      - [x] See this issue and see if there are any issues on other repositories that might be required to complete the dashboard.
   - Partners
      - [x] Change title to be Current and Past Partners
   - Sponsors
      - [x] Change title to be Current and Past Sponsors

### Wins
- URL: https://hackforla.org/wins
- [x] [Wins issues on the project board](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22p-feature%3A+wins+page%22)
   - [x] Write an issue to manual pick a new win, and hard code it on the about page.  The randomizer can't get worked on until the homepage branch is merged into the gh-pages branch
      - [ ] ~https://github.com/hackforla/website/issues/4282 (has a dependency on 4910 being finished)~
      - [ ] ~#4190~
   - [x] Create an issue to change the WINS form to only offer to use their GitHub image.  We will not be able to display people's LinkedIn profile pictures, the [LinkedIn API Terms of Service](https://www.linkedin.com/legal/l/api-terms-of-use) disallows it. So we need to revise the form to reflect that.
   - [x] #6272
   - [x] #3140 
   - [x] #4388 


### Join Us
-  [Join us page Issues on the project board](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22p-feature%3A+Join+Page%22)
- URL: https://www.hackforla.org/join
   - [x] https://github.com/hackforla/website/issues/5973

### Program Areas
- URL: https://www.hackforla.org/program-areas
- [Program Areas Issues on the project board](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22p-feature%3A+program+area%22)
   - [x] https://github.com/hackforla/website/issues/3328


### Projects Page
- [x] URL: https://www.hackforla.org/projects
- [x] [Projects page Issues on the project board](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22p-feature%3A+projects+page%22)
  - The drop downs have to have consistent content in them (problems with languages, technologies and tools).  
     - https://github.com/hackforla/website/issues/6709
     - https://github.com/hackforla/website/issues/4434
     - https://github.com/hackforla/website/issues/4262
        - Make an ER to address the difference between
           - https://www.hackforla.org/projects-check/?technologies=Docker
           - https://www.hackforla.org/projects-check/?languages=Dockerfile
  - They need to look better.  The chevron we currently have is unattractive.  See http://civictechindex.org/about/faq and make an ER to improve ours


### Project Pages
- Access the Data
   - [x] https://github.com/hackforla/website/issues/5127
   - [x] https://github.com/hackforla/website/issues/6138
   - [x] https://github.com/hackforla/website/issues/6144
   - [x] https://github.com/hackforla/website/issues/5337
- Expunge Assist
   - [x] https://github.com/hackforla/website/issues/6834
   - [x] https://github.com/hackforla/website/issues/6866
   - [x] https://github.com/hackforla/website/issues/6579
- New Project
   - [x] https://github.com/hackforla/website/issues/4342
- All project pages
   - [x] https://github.com/hackforla/website/issues/6838
   - [x] https://github.com/hackforla/website/issues/6930
   - [x] https://github.com/hackforla/website/issues/6129
   - [x] https://github.com/hackforla/website/issues/6930
   - [x] https://github.com/hackforla/website/issues/6920
   - [x] https://github.com/hackforla/website/issues/6655

### CoPs
- [x] URL: https://www.hackforla.org/communities-of-practice
- [x] [Communities of Practice Issues on the project board](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22p-feature%3A+communities+of+practice%22)
  - [x] Needs to be reviewed to see if updates are required

### Events
- URL: https://www.hackforla.org/events
- [x] [Events Issues on the project board](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22p-feature%3A+events%22)
- [x] https://github.com/hackforla/website/issues/6320 (this issue is also relevant to Project info and Project meetings pages
- [x] https://github.com/hackforla/website/issues/6749
- [x] https://github.com/hackforla/website/issues/6369

### Toolkit
- URL: https://www.hackforla.org/toolkit
- [x] Remove the toolkit from the menu.  Bring it back when its ready

### Sitemap
- URL: https://www.hackforla.org/sitemap
- [x] find or make new issue to update sitemap

### On many pages
#### SDGs
- [SDG issues on project Board](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22p-feature%3A+sdgs%22)
- Issues
   - [x] Evaluate the [UN guidelines for SDG usage](https://www.un.org/sustainabledevelopment/wp-content/uploads/2019/01/SDG_Guidelines_AUG_2019_Final.pdf) to determine if we can or cannot round edges of the SDGs.  
      - [x] If we cannot, then we need to update the citizen engagement page to have squared corners like this [figma: citizen engagement](https://www.figma.com/design/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=17601-123536&t=z8eB1YcqpTS5xfkq-0)
      - [x] if we can, we need to evaluate if we want to change the 
           - [x] [About page Sustainability section](https://www.hackforla.org/about/#sustainability)
           - [x] [Citizen Engagement page](https://www.hackforla.org/citizen-engagement)
   - [x] https://github.com/hackforla/website/issues/4229
   - [x] https://github.com/hackforla/website/issues/2037
   - [x] https://github.com/hackforla/website/issues/6277
   - [x] https://github.com/hackforla/website/issues/4153
      - [x] https://github.com/hackforla/website/issues/6223
      - [x] https://github.com/hackforla/website/issues/6222
      - [x] https://github.com/hackforla/website/issues/6221
      - [x] https://github.com/hackforla/website/issues/6220
- Resources
   - [Program Area Audit Spreadsheet](https://docs.google.com/spreadsheets/d/1wkzmWck-aGTzGxiryYEUhGbKRJX8rDDT3RLsAn-E-9o/edit#gid=0) - has the SDG info for all projects and issue tracking

### Sitewide
- [x] https://github.com/hackforla/website/issues/6140
- [x] https://github.com/hackforla/website/issues/6161
- all spelling issues
   - [x] https://github.com/hackforla/website/issues/5248 see this epic issue for details.
   - [x] https://github.com/hackforla/website/issues/5450
- [x] confirm that all links on the header and footer work
   - [x] https://github.com/hackforla/website/issues/2455

### Additional links
<details><summary>Homepage image</summary>
<p>

![Group 737](https://github.com/hackforla/website/assets/37763229/7c72b1c2-e7f6-4494-9834-3e7b9951ffac)


</p>
</details>

**Comment by ExperimentsInHonesty** [comment]
CTJ Accomplishments
- https://github.com/hackforla/CivicTechJobs/issues/548

**Comment by HackforLABot** [comment]
@yugmamoradia, @ExperimentsInHonesty

Please add update using the below template (even if you have a pull request). Afterwards, remove the '2 weeks inactive' label and add the 'Status: Updated' label.
1. Progress: "What is the current status of your project? What have you completed and what is left to do?"
2. Blockers: "Difficulties or errors encountered."
3. Availability: "How much time will you have this week to work on this issue?"
4. ETA: "When do you expect this issue to be completed?"
5. Pictures (optional): "Add any pictures of the visual changes made to the site so far."

If you need help, be sure to either: 1) place your issue in the `Questions/In Review` column of the Project Board and ask for help at your next meeting, 2) put a "Status: Help Wanted" label on your issue and pull request, or 3) put up a request for assistance on the #hfla-site channel. Please note that including your questions in the issue comments- along with screenshots, if applicable- will help us to help you. [Here](https://github.com/hackforla/website/issues/1619#issuecomment-897315561) and [here](https://github.com/hackforla/website/issues/1908#issuecomment-877908152) are examples of well-formed questions.

<sub>You are receiving this comment because your last comment was before Tuesday, August 27, 2024 at 12:04 AM PST.</sub>

**Comment by HackforLABot** [comment]
@yugmamoradia, @ExperimentsInHonesty

Please add update using the below template (even if you have a pull request). Afterwards, remove the '2 weeks inactive' label and add the 'Status: Updated' label.
1. Progress: "What is the current status of your project? What have you completed and what is left to do?"
2. Blockers: "Difficulties or errors encountered."
3. Availability: "How much time will you have this week to work on this issue?"
4. ETA: "When do you expect this issue to be completed?"
5. Pictures (optional): "Add any pictures of the visual changes made to the site so far."

If you need help, be sure to either: 1) place your issue in the `Questions/In Review` column of the Project Board and ask for help at your next meeting, 2) put a "Status: Help Wanted" label on your issue and pull request, or 3) put up a request for assistance on the #hfla-site channel. Please note that including your questions in the issue comments- along with screenshots, if applicable- will help us to help you. [Here](https://github.com/hackforla/website/issues/1619#issuecomment-897315561) and [here](https://github.com/hackforla/website/issues/1908#issuecomment-877908152) are examples of well-formed questions.

<sub>You are receiving this comment because your last comment was before Tuesday, September 3, 2024 at 12:04 AM PST.</sub>

**Comment by HackforLABot** [comment]
@yugmamoradia, @ExperimentsInHonesty

Please add update using the below template (even if you have a pull request). Afterwards, remove the '2 weeks inactive' label and add the 'Status: Updated' label.
1. Progress: "What is the current status of your project? What have you completed and what is left to do?"
2. Blockers: "Difficulties or errors encountered."
3. Availability: "How much time will you have this week to work on this issue?"
4. ETA: "When do you expect this issue to be completed?"
5. Pictures (optional): "Add any pictures of the visual changes made to the site so far."

If you need help, be sure to either: 1) place your issue in the `Questions/In Review` column of the Project Board and ask for help at your next meeting, 2) put a "Status: Help Wanted" label on your issue and pull request, or 3) put up a request for assistance on the #hfla-site channel. Please note that including your questions in the issue comments- along with screenshots, if applicable- will help us to help you. [Here](https://github.com/hackforla/website/issues/1619#issuecomment-897315561) and [here](https://github.com/hackforla/website/issues/1908#issuecomment-877908152) are examples of well-formed questions.

<sub>You are receiving this comment because your last comment was before Tuesday, September 10, 2024 at 12:04 AM PST.</sub>

**Comment by ExperimentsInHonesty** [comment]
We have made some issues for projects to give us their accomplishments for the accomplishment page.  We have created a new project board to track that [Admin: Project Team Tasks, Accomplishment tab](https://github.com/orgs/hackforla/projects/7/views/10)

We need to find all the relevant issues for each project and add them to the board .  Here is an org search for accomplishment https://github.com/search?q=org%3Ahackforla+accomplishments&type=issues

**Comment by HackforLABot** [comment]
@ExperimentsInHonesty

Please add update using the below template (even if you have a pull request). Afterwards, remove the '2 weeks inactive' label and add the 'Status: Updated' label.
1. Progress: "What is the current status of your project? What have you completed and what is left to do?"
2. Blockers: "Difficulties or errors encountered."
3. Availability: "How much time will you have this week to work on this issue?"
4. ETA: "When do you expect this issue to be completed?"
5. Pictures (optional): "Add any pictures of the visual changes made to the site so far."

If you need help, be sure to either: 1) place your issue in the `Questions/In Review` column of the Project Board and ask for help at your next meeting, 2) put a "Status: Help Wanted" label on your issue and pull request, or 3) put up a request for assistance on the #hfla-site channel. Please note that including your questions in the issue comments- along with screenshots, if applicable- will help us to help you. [Here](https://github.com/hackforla/website/issues/1619#issuecomment-897315561) and [here](https://github.com/hackforla/website/issues/1908#issuecomment-877908152) are examples of well-formed questions.

<sub>You are receiving this comment because your last comment was before Tuesday, October 8, 2024 at 12:05 AM PST.</sub>

**Comment by HackforLABot** [comment]
@ExperimentsInHonesty

Please add update using the below template (even if you have a pull request). Afterwards, remove the '2 weeks inactive' label and add the 'Status: Updated' label.
1. Progress: "What is the current status of your project? What have you completed and what is left to do?"
2. Blockers: "Difficulties or errors encountered."
3. Availability: "How much time will you have this week to work on this issue?"
4. ETA: "When do you expect this issue to be completed?"
5. Pictures (optional): "Add any pictures of the visual changes made to the site so far."

If you need help, be sure to either: 1) place your issue in the `Questions/In Review` column of the Project Board and ask for help at your next meeting, 2) put a "Status: Help Wanted" label on your issue and pull request, or 3) put up a request for assistance on the #hfla-site channel. Please note that including your questions in the issue comments- along with screenshots, if applicable- will help us to help you. [Here](https://github.com/hackforla/website/issues/1619#issuecomment-897315561) and [here](https://github.com/hackforla/website/issues/1908#issuecomment-877908152) are examples of well-formed questions.

<sub>You are receiving this comment because your last comment was before Tuesday, October 15, 2024 at 12:04 AM PST.</sub>

**Comment by ExperimentsInHonesty** [comment]
We have a new PM on the project who will be owning this launch

**Comment by ExperimentsInHonesty** [comment]
- [x] Create a template for our page epics
   - [x] copy it
      ```
      ### Overview
      We need to track existing issues and make new issues to deliver [REPLACE WITH PAGE NAME] complete enough for the Homepage launch

      #### Details
      We need to track and support the final wrap up items for the homepage launch, so that it can be completed.

      ### Action Items
      - [ ] List all the issues related to the label
      - [ ] Identify what open issues have to be resolved before launch
      - [ ] Identify items that need issues opened

      ### Resources/Instructions
      - Parent Epic: https://github.com/hackforla/website/issues/2481
      
      #### Used for creating this issue
      - 1.01 Label name: 
      - 1.02 URL of page on Hack for LA website:
      - 1.03 URL of link to code:

      #### Created by this issue
      - 2.01 [EXAMPLE would be a link to an issue related to delivering this page]
      ```
- [x] Create new EPIC issues
   - [x] customize each with content from top of issue and this comment https://github.com/hackforla/website/issues/2481#issuecomment-2155756054
   - [x] customize each with labels from the [p-feature label series](https://github.com/hackforla/website/labels?q=p-feature%3A)
   - [x] review with PM (bonnie)
- [ ] Prioritize the epics

**Comment by ExperimentsInHonesty** [comment]
- [x] Make an epic for https://www.hackforla.org/project-meetings

- https://github.com/hackforla/website/issues/7904

**Comment by Rabia2219** [comment]
#### Tasks
Go through all the sub issue epics and their child issues and do the following tasks

---

- [x] Add the following check boxes under #Action Items
   ```
   - [ ] Make the issues specified
   - [ ] Link them as sub issues to this issue
   ```

---


- [x] update the resource 1.01 Label Name to match the issue https://github.com/hackforla/website/issues/7664 as shown below
```
- 1.01 Label name: P-Feature: Wins Page
    - 1.01.01 Issue Tab: [P-Feature: Wins Page](https://github.com/hackforla/website/labels/P-Feature%3A%20Wins%20Page)
    - 1.01.02 Project Board: [P-Feature: Wins Page](https://github.com/orgs/hackforla/projects/86/views/1?filterQuery=+label%3A%22P-Feature%3A+Wins+Page%22)
```
---

- [ ] update 1.03
   - [ ] update the text at 1.03 with the following link text, but replace WINS with the name of the page the issue is about
   ```
   - 1.03 URL of link to code search: [repo:hackforla/website NOT language:SVG NOT path:/^_data/internal/credits// WINS]()
   ```
   - [ ]  go to [repo search page with repo:hackforla/website NOT language:SVG NOT path:/^_data\/internal\/credits\//](https://github.com/search?q=repo%3Ahackforla%2Fwebsite+NOT+language%3ASVG+NOT+path%3A%2F%5E_data%5C%2Finternal%5C%2Fcredits%5C%2F%2F&type=code) 
      - [x] add a space to the end of the search string into the search bar
      - [ ] add the name of your page after the space
      - [x] hit the enter key
   - [x] copy the URL, and paste it into the empty parenthesis on 1.03
   - [x] hit the save button

---

- [ ] Add WIKI resources to issue
   - [ ] Add this text to the resource section
      ```
      - 1.04 URL of WIKI page: 
      ```
   - [ ] Go to [repo search page with repo:hackforla/website PAGE and wikis](https://github.com/search?q=repo%3Ahackforla%2Fwebsite+PAGE&type=wikis)
      - [ ] Replace the word PAGE with the name of your page
      - [ ] hit the enter key
      - [ ] copy the URL, and paste it after the `:` on 1.04
   - [ ] Add this text under 1.04
      ```
         - 1.04.01 URL of future edits to wiki pages: https://github.com/hackforla/website/wiki/How-to-Contribute-to-the-Wiki (check this page to see if there are any other new edits in the queue.)
      ```
   - [ ] hit the save button

**Comment by ExperimentsInHonesty** [comment]
- [ ] Add this template to the bottom of the top portion of issue


### Template to use
````
---
## Issues to write
### Must Have

### Should Have

### Could Have

---
## Relevance of the issues to the Homepage launch
### Must Have
- #[ADD ISSUE NUMBER]
   - why: 
   - status: 
      - draft
      - responsible: 
   - role: 
### Should Have
- #[ADD ISSUE NUMBER]
   - why: 
   - status: 
      - draft
      - responsible: 
   - role: 
### Could Have
- #[ADD ISSUE NUMBER]
   - why: 
   - status: 
      - draft
      - responsible: 
   - role: 

### Won't have this time
<details><summary>List of all won't have issues</summary>

- #
- #
- #
- #
- #
- #
</details> 


#### Templates
```
- # 
   - why: 
   - status: 
      - draft
      - responsible: 
   - role:  
```
```
### Issues list associated with this epic
last updated: 
Highest number found:
```

<details><summary>List of all issue by number</summary>

- [ ] #
- [ ] #
</details> 
````

Example from 
- https://github.com/hackforla/website/issues/7664
   >### Must Have
   >- #7594
   >   - why: if wins don't show up in the new project board, they won't be reviewed and added to the site
   >   - status: new issue approval
   >      - draft
   >      - responsible: RW
   >   - role: backend/devOps

**Comment by ExperimentsInHonesty** [comment]
once the tasks in these issue comments are done for a sub issue under this epic
- https://github.com/hackforla/website/issues/2481#issuecomment-2654386735
- https://github.com/hackforla/website/issues/2481#issuecomment-2654544272

- [x] remove the draft label 
- [ ] add the ready for prioritization label

**Comment by Rabia2219** [comment]
@priyanka02art 
Hi Priyanka,
I confirmed with Bonnie that [these task steps](https://github.com/hackforla/website/issues/2481#issuecomment-2654386735) should be followed as is. There is no change to the instructions for 1.04, which you had asked clarification for.

**Comment by ExperimentsInHonesty** [comment]
@priyanka02art Please provide update
1. Progress: "What is the current status of your project? What have you completed and what is left to do?"
2. Blockers: "Difficulties or errors encountered."
3. Availability: "How much time will you have this week to work on this issue?"
4. ETA: "When do you expect this issue to be completed?"
5. Pictures or links* (if necessary): "Add any pictures or links that will help illustrate what you are working on."

* remember to add links to the top of the issue if they are going to be needed again.

**Comment by ExperimentsInHonesty** [comment]
- [x] update the resource 1.01 Label Name to match the issue https://github.com/hackforla/website/issues/7664 as shown below
```
- 1.01 Label name: P-Feature: Wins Page
    - 1.01.01 Issue Tab: [P-Feature: Wins Page](https://github.com/hackforla/website/labels/P-Feature%3A%20Wins%20Page)
    - 1.01.02 Project Board: [P-Feature: Wins Page](https://github.com/orgs/hackforla/projects/86/views/1?filterQuery=+label%3A%22P-Feature%3A+Wins+Page%22)
```

Issues that need to be done
- [x] https://github.com/hackforla/website/issues/7667
- [x] https://github.com/hackforla/website/issues/7670
- [x] https://github.com/hackforla/website/issues/7672
- [x] https://github.com/hackforla/website/issues/7673
- [x] https://github.com/hackforla/website/issues/7674
- [x] https://github.com/hackforla/website/issues/7675
- [x] https://github.com/hackforla/website/issues/7676
- [x] https://github.com/hackforla/website/issues/7679
- [x] https://github.com/hackforla/website/issues/7680
- [x] https://github.com/hackforla/website/issues/7904

**Comment by ExperimentsInHonesty** [comment]
```
### Issues list associated with this epic
last updated: 
Highest number found:
```

**Comment by ExperimentsInHonesty** [comment]
### Roadmap
- Create Epics for each page on site
- Gather related issues
- Classify issues
- [ ] Review a single epic, and determine if all the issues needed for the epic have been created
   - [ ] if it needs issues 
      - [ ] define new issues to create
      - [ ] create issues
   - [ ] if epic has all issues
      - [ ] review the status of the issues
         - [ ] resolve blockers
         - [ ] define monitoring cadence
      - [ ] estimate time to completion

**Comment by ExperimentsInHonesty** [comment]
### Add this to the bottom of the action items

```
   - [ ] if epic has all issues
      - [ ] review the status of the issues
         - [ ] resolve blockers
         - [ ] define monitoring cadence
      - [ ] estimate time to completion
```

**Comment by ExperimentsInHonesty** [comment]
We need to update the Epics for pages with the following, while keeping any of the check marks for Action items that are completed.

### Replace Action Items
Before
```
### Action Items
- [ ] List all the issues related to the label
- [ ] Identify what open issues have to be resolved before launch
- [ ] Identify items that need issues opened
- [ ] Make the issues specified
- [ ] Link them as sub issues to this issue
``` 

After
````
### Action Items
- [ ] List all the issues related to the label
- [ ] Identify what open issues have to be resolved before launch, by classifying the issues into the following categories using MoSCoW method in a comment.
   - Must Have
   - Should Have
   - Could Have
   - Won't Have this time
- [ ] Add the following details under each issue
   ```
   - why: [ADD WHY ITS IMPORTANT TO TO THE LAUNCH]
   - status: [ADD STATUS e.g., Icebox, Emergent Requests, New Issue Approval, ERs and epics that are ready to be turned into issues, Prioritized Backlog, In Progress (actively working), Questions / In Review]
      - draft: [true OR false]
      - responsible: [ADD GITHUB HANDLES WITHOUT @]
   - role: [ADD ROLE e.g. data scientist, infrastructure, design, design lead, user research, Research Lead, dev leads, data analyst, BA, hfla leadership, back end/devOps, writing, front end, merge team, technical writer]
   ```
- [ ] Once you receive sign-off from Staff PM add to `## Relevance of the issues to the Homepage launch`
- [ ] Review comments with `Notes carried forward from epic` if it exists, and make sure all previous items are addressed in the top of the issue.
   - If it specifies new issues to make then add explicit action items for making those issue in a new comment on this issue
      - get sign-off on those new issues to make by Staff PM 
- [ ] Identify items that need issues opened by looking at the page and features and seeing what is not already addressed by the existing issues or new issues to make list.
- [ ] Make the issues specified
- [ ] Link them as sub issues to this issue
- [ ] Classify them using MoSCoW and add the details (why, status, draft, responsible, role)
- [ ] Get Sign-off and prioritization of the issues by Staff PM
- [ ] If this page has content that must be checked for accuracy before homepage launch, add the detail to the epic for tracking that (resource 1.05)
````

### Add New Resource item 
```
- 1.05 Issue for adding items to a final checklist: #8417
```

**Comment by nadiaspn0503** [comment]
Make an EPIC for the Donation page and add this issue to write
- [ ] Update civic tech structure in Make a Donation section from `Civic Tech Structure` to `Civic Tech Structure, Inc.`

**Comment by ExperimentsInHonesty** [comment]
Make three new labels
HLC: M
HLC: S
HLC: C

change labels on each Homepage Launch Countdown to the new appropriate label

**Comment by kimberlyvaldez07** [comment]
2026/08/04

The three new labels have been created:
HLC: M
HLC: S
HLC: C

Each issue has been labeled with the appropriate label.
